pkg/sdn/api/v1/types.go

```diff
@@ -11,19 +11,21 @@ const (
 
 // +genclient=true
 
-// ClusterNetwork describes a cluster network
+// ClusterNetwork describes the cluster network. There is normally only one object of this type,
+// named "default", which is created by the SDN network plugin based on the master configuration
+// when the cluster is brought up for the first time.
 type ClusterNetwork struct {
 	unversioned.TypeMeta `json:",inline"`
 	// Standard object's metadata.
 	kapi.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
-	// Network is a CIDR string to specify the global overlay network's L3 space
+	// network is a CIDR string specifying the global overlay network's L3 space
```

Comment:
This Godoc is turned into swagger, which is read by humans in the context of JSON responses.   Someone reading swagger will never see the Go types, they will only ever see the JSON responses.  Returning the Go type name is thus not useful to them, while the JSON field name is.